### ISSUE TITLE
0x%p Prefixing 0x with %p formatting might not be necessary

### ISSUE THREAD

**Issue by SebMenozzi** [orangeduck/Cello#143]
Small detail here related to `%p` formatting.
After watching this video of @tsoding showcasing Cello  => [Link of the video here](https://www.youtube.com/watch?v=SbpusG7sHF4&t=2513s), 
He noticed that `0x` with `%p` might be redundant depending the compiler implem

**Comment by sincyn** [comment]
Not my project, but why don't you just fix it yourself if you care enough to make an issue? Should be more than a function or two to fix.

**Comment by SebMenozzi** [comment]
@sincyn yes I could "fix" it with another macro, I'm not even using Cello in any of my projects (yet), I just wanted to let the current/future maintainer(s) know about it (if this cool project is being updated/forked), not a big deal...
